src/lib/utils.js

```diff
@@ -0,0 +1,18 @@
+import { debounce, throttle } from 'lodash';
+
+export const isFunction = fn => typeof fn === 'function';
+
+export const listHandle = {
+  debounce,
+  throttle,
+};
+
+export const getHandle = (type) => {
+  const handle = listHandle[type];
+  if (!isFunction(handle)) {
+    return false;
+  }
+  return handle;
+};
+
+export const checkIsHaveWindow = () => typeof window !== 'undefined';
```

Comment:
maybe it's better to rename this function to `isInBrowser` or `isBrowser`

Reply:
yep, I will change it

Reply:
I think to use `checkIsHaveWindow` will clearly in cases use in browser or test.